mule-extensions-api/src/main/java/org/mule/runtime/extension/api/runtime/Interceptor.java

```diff
@@ -6,6 +6,8 @@
  */
 package org.mule.runtime.extension.api.runtime;
 
+import org.mule.runtime.extension.api.runtime.operation.OperationContext;
```

Comment:
Isn't this an operation interceptor? Should it be part of the operation package as well?